docs-client/src/containers/MethodPage/index.tsx

```diff
@@ -129,6 +132,8 @@ type Props = OwnProps &
   }>;
 
 const MethodPage: React.FunctionComponent<Props> = (props) => {
+  const [debugFormIsOpen, setDebugFormIsOpen] = React.useState(false);
```

Comment:
It would be better to open debug form by default if the form is filled with preset values with the URL parameters.

Reply:
Deleted `scrollToDebugForm` and instead called `setDebugFormIsOpen(true)` if url params existed.

Reply:
Sounds good.